include/seqan/basic/debug_test_system.h

```diff
@@ -164,7 +164,18 @@
 #else  // #ifdef STDLIB_VS
 #include <unistd.h>     // unlink()
 #include <sys/stat.h>   // mkdir()
+// gcc 9.3 -std=c++2a has ignored attribute warnings on macOS
+#if defined(__GNUC__) && defined(__APPLE__) && (__GNUC__ == 9) && __cplusplus > 201703L
+#define unused_backup __unused 
+#undef __unused
+#define __unused 
 #include <dirent.h>     // DIR
+#undef __unused
+#define __unused unused_backup
+#undef unused_backup
```

Comment:
this does not seem to work:

https://godbolt.org/z/75aT9M

I found this answer:
https://stackoverflow.com/a/1545079

This seems to work:
https://godbolt.org/z/TMd3f9

Reply:
It somehow worked, even though it shouldn't :)